web/client/components/map/openlayers/Map.jsx

```diff
@@ -242,6 +244,21 @@ class OpenlayersMap extends React.Component {
                 this.map.removeControl(this.map.getControls().getArray().filter((ctl) => ctl instanceof Zoom)[0]);
             }
         }
+
+        if (!isEqual(newProps.bbox, this.props.bbox) && newProps.bbox && !isEmpty(newProps.bbox) && newProps.zoomToExtent) {
```

Comment:
That would require quite some time, and i noticed that supports are bound to a piece of global state like
[state.draw](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/map/index.js#L74) or state.measurement
Another problem that is present even with support is the dependenciesToExtent that actually are causing unnecessary re-renders especially for preview map

Reply:
> unfortunately we can't do this because it can cause issues for all the maps that update their state

no, only the ones that has some dependencies connected to a parent table

Reply:
please, checkout new solution if it can be good too.